### DISCUSSION TITLE
API Question

### DISCUSSION THREAD

**Discussion by devPhases** [oobabooga/text-generation-webui#4824] [Q&A]
I was trying to follow an example on this page, https://github.com/oobabooga/text-generation-webui/wiki/12-%E2%80%90-OpenAI-API

It shows how to change the api port "To change the port, which is 5000 by default, use --api-port 1234 (change 1234 to your desired port number)."

I entered "./start_linux.sh --api-port 5001" but it does not seem to create that port and/or api.

```
mike@mike-GL63-9SDK:/media/mike/70c75d3e-9832-4991-b0e6-414481a4b934/text-generation-webui-main$ ./start_linux.sh --api-port 5001
2023-12-05 21:16:27 INFO:Loading the extension "gallery"...
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
```

I went to "http://127.0.0.1:5001/" to check and it says "unable to connect"

I'm wondering if this is a bug  or maybe the information is outdated?

Some additional context: I run multiple instances of webui. That works fine when not adding the --api on startup. I wanted to create an api for each instance. The first run does create the 5000 port. The other instances wouldn't create one, and I found that was because they were trying to use 5000 also. So I closed all of the instances, restarted my computer just in case there was something odd going on with the ports, and on the first webui run, i tried to create api on port 5001.

**Comment by taz4et** [comment]
./start_linux.sh --api --api-port 5001
